temoa/model_checking/commodity_graph.py

```
@@ -1,261 +1,290 @@
+    )
+
+    # 5. Define the style legend data
+    style_legend_map = [
+        {'label': 'Connected to Demand Orphan', 'borderColor': '#d62728', 'borderWidth': 4},
+        {'label': 'Connected to Other Orphan', 'borderColor': '#ff7f0e', 'borderWidth': 4},
+        {'label': 'Connected to Driven Tech', 'borderColor': '#1f77b4', 'borderWidth': 4},
+    ]
+
+    # 6. Create the interactive HTML visualization
+    output_file = config.output_path / f'Network_Graph_{region}_{period}.html'
+    unique_sectors = sorted(list(sector_colors.keys()))
```

Ruff (0.14.1)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)